### ISSUE TITLE
CodeFix error, when using vue-apollo

### ISSUE THREAD

**Issue by academici** [universal-vue/uvue#21]
**Describe the bug**
```js
 ERROR  TypeError: Cannot read property 'plugins' of undefined
TypeError: Cannot read property 'plugins' of undefined
    at CodeFixer.run (/home/vostrikov/apps/autosalon/node_modules/@uvue/vue-cli-plugin-ssr/uvue/CodeFixer.js:97:20)
    at process._tickCallback (internal/process/next_tick.js:68:7)
error Command failed with exit code 1.
```

**To Reproduce**
Create vue cli project
Add vue apollo plugin (vue add apollo) without code examples and server
Add uvue plugin
On code fix step catch error

Please indicate versions of:
```
node: 
vue-cli: 3.3.0
uvue: v0.1.0-alpha.15
os: ubuntu 18.04
browser: chrome 71
```

**Comment by chymz** [comment]
Nice catch, I'll fix this soon

**Comment by chymz** [comment]
Fixed in `0.1.0-alpha.16`
